af/shovel/oonipl/metrics.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+
+"""
+Metric generation
+"""
+
+from os.path import basename, splitext
+
+import statsd
+
+
+def setup_metrics(host="localhost", name=None):
+    """Setup metric generation. Use dotted namespaces e.g.
+    "pipeline.centrifugation"
```

Comment:
Maybe you can `assert` or `throw` if a dotted namespace is important?

Reply:
The dotted structure is for our convenience to create a hierarchy of metrics. If a component is clearly identified by a single word we can use it without dots.